src/Workspaces/Core/Desktop/Workspaces.Desktop.csproj

```diff
@@ -31,15 +31,6 @@
     <DefineConstants>$(DefineConstants);MSBUILD12</DefineConstants>
```

Comment:
And this is also casting off the ability to do the MSBUILD12 compilation. Want to delete these lines and delete the code that was #if MSBUILD12 too while you're here?